src/Button/index.js

```diff
@@ -471,12 +474,18 @@ const Button = React.forwardRef((props: Props, ref: Ref) => {
     loading = false,
     width = 0,
     role,
+    title,
   } = props;
   const iconLeft = props.iconLeft || icon;
   const sizeIcon = size === ICON_SIZES.SMALL ? ICON_SIZES.SMALL : ICON_SIZES.MEDIUM;
   const onlyIcon = iconLeft && !children;
   const isDisabled = loading || disabled;
 
+  warning(
+    !(!children && !title),
+    "Warning: title is missing on Button. Use title prop to add aria-label to be accessible for screen readers",
```

Comment:
I would use:
```
Warning: children or title property is missing on Button. Use title property to add aria-label to be accessible for screen readers.
```
We can also append a link for documentation or README to the warning, if you want 🙂